python/test/t_GaussianNonLinearCalibration_std.expout

```diff
@@ -4,3 +4,4 @@ result (Global)= [2,1.44,0.48]
 result   (Auto)= [2.61,1.22,0.498]
 result    (TNC)= [2.61,1.22,0.498]
 result (Global)= [2,1.44,0.48]
+result (unobs.)= [2.8,2.18,1.5,1.85]
```

Comment:
```suggestion
result (unobs.)= [1.64,2.5,0.644,0.871]
```
This other result is consistent with the change to `t_GaussianNonLinearCalibration_std.py` I propose below.